### DISCUSSION TITLE
No repositories, jobs, graphs, pipeline found when deploying via github actions -- but everything works fine locally

### DISCUSSION THREAD

**Discussion by nlisauskas** [dagster-io/dagster#18841] [Community]
Hi Dagster team and users! I'm trying to deploy a serverless project via github actions for the first time and everything works properly locally, but can't for the life of me get the deployment to succeed on dagster cloud... any thoughts? The photos below are my directory structure and error message, respectively.

![image](https://github.com/dagster-io/dagster/assets/20713538/67a90d1c-6812-4f75-9224-26bf82b2a087)
![image](https://github.com/dagster-io/dagster/assets/20713538/4c0c2643-d771-4eae-aba2-668e3473577e)

**Comment by prha** [comment]
@nlisauskas do you have a `Definitions` object in your code location?  

Can you confirm that this is working locally?  For local deployments, we use a `workspace.yaml` file to point your local instance to your code locations.  In cloud, we use a `dagster_cloud.yaml` to tell the GitHub actions script to build containers from your git repo, but rely on a `Definitions` object in the built package (probably exported in your `__init__.py` file).

If it is working locally, it might be worth seeing the differences in your `workspace.yaml` / `dagster_cloud.yaml` files.
